NoRootNamesError: No source files found

I just updated from version 0.95.0 to 0.96.0 and the schema generator stopped working, I now get this message:

```
> ts-json-schema-generator -f tsconfig.json -t Foo > foo.schema.json

NoRootNamesError: No source files found
```

Can be reproduced with a minimal tsconfig.json (even one containing an empty object) and a simple `export interface Foo { bar: string }` type. Works with schema generator 0.95.0, but not with 0.96.0

Was there a change in the CLI parameters? I can't find any entry in the changelog which sounds like a CLI change.

And maybe this issue is a good point to discuss the versioning of the the schema-generator. I'm using it for a long time now and I would say the interfaces for which I generate JSON schemas are very complex (Lots of generic types with many mapped types and conditional types for example). And since my last contributions to this project two years ago I never had a problem with it (well, until now). So in my opinion the schema generator is very stable and earns the right to get a production version number. After this the versioning should follow the semver paradigm and the major version number should be incremented when the public API (including the CLI parameters) is no longer compatible to the previous version.

Hmm, looks like we don't resolve paths correctly in the first place.

```
$ yarn run run -f ./node_modules/vega-lite/tsconfig.json -t AggregateOp
error TS6053: File 'node_modules/vega-lite/node_modules/vega-lite/src/index.ts' not found.
  The file is in the program because:
    Root file specified for compilation
```

I fixed that in https://github.com/vega/ts-json-schema-generator/pull/979.

Could you try the version from the pr?

--- a/factory/program.ts
+++ b/factory/program.ts
@@ -22,7 +22,7 @@ function loadTsConfigFile(configFile: string) {
         const parseResult = ts.parseJsonConfigFileContent(
             config.config,
             ts.sys,
-            path.dirname(configFile),
+            path.resolve(path.dirname(configFile)),
             {},
             configFile
         );

